pandas/parser.pyx

```diff
@@ -1239,6 +1239,10 @@ class CParserError(ValueError):
     pass
 
 
+class EmptyDataError(ValueError):
+    pass
```

Comment:
I would move both of these (as well as the ``ParserWarning``) to ``io/common.py`` (you *may* need to import them when you are using to avoid circular imports)